docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.7'
+version: '2.0'
```

Comment:
Been getting. Changing to 2.0 fixes it, I'll change it back to 3.7 for the sake of the PR.
`ERROR: Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a version of "2" (or "2.0")`

Reply:
I think you might need to pull down a newer version of `docker-compose` and/or `docker`